tpm/internal/key/key.go

```diff
@@ -0,0 +1,267 @@
+//nolint:errorlint,revive // copied code from github.com/google/go-attestation
```

Comment:
since this is copied code add this to the header:

`CODE COPIED FROM ...; DO NOT EDIT!`